pyreisejl/utility/launchers.py

```diff
@@ -22,11 +24,28 @@ class Launcher:
         where HH, MM, and SS are optional.
     :param int interval: length of each interval in hours
     :param str input_dir: directory with input data
+    :param None/str execute_dir: directory for execute data. None defaults to an
```

Comment:
General question: do we need to document `None` as the type? Since anything can be None, I wonder if it should be left to the description (if there is a default) or `raises` if we reject it, or even omitted if it just works in an intuitive way.

Reply:
I usually don't list None in the param type description but mention it in the doc string if it is the default.

Reply:
I either omit it or as @BainanXia does mention it in the description

Reply:
Done.